ticdc: update the docs about create changefeed (#3985)
pingcap/docs: releases/release-4.0.6.md

+ Tools

>     + TiCDC (TiCDC is **a feature for general availability** since v4.0.6. You can use it in the production environment.)

Reviewer: ```suggestion
    + TiCDC (GA since v4.0.6 and available for production environments)
```

>     + TiCDC (GA since v4.0.6 and available for production environments)

Reviewer: ```suggestion
    + TiCDC (GA since v4.0.6)
```
